contracts/token/BiconomyTokenPaymaster.sol

```diff
@@ -259,14 +293,14 @@ contract BiconomyTokenPaymaster is BasePaymaster, ReentrancyGuard, TokenPaymaste
      * @param dest address to send to
      */
     function withdrawAllETH(address dest) public onlyOwner nonReentrant {
```

Comment:
Duplicate method `withdrawAllETHTo` found which is doing the same thing.

Reply:
nice catch. removed it